docs/BaremetalHost_ProvisioningState.dot

```diff
@@ -1,5 +1,7 @@
 digraph BaremetalHost {
-    None -> Registering
+    None -> Discovered [label="(without BMC access details)"]
+    Discovered -> Registering [label="set BMC access details"]
```

Comment:
I think this could just be label=done for consistency.

Reply:
I changed it to the same code snippet that I used earlier.